src/DateTimeFormatter.php

```diff
@@ -16,10 +16,22 @@ class DateTimeFormatter extends AbstractFilter
     /**
      * A valid format string accepted by date()
      *
+     * @todo This default should be changed to DateTime::ATOM or perhaps DateTime::RFC3339
```

Comment:
Please remove this comment and open a separate issue report or pull request for this to-do. Each comment in the code is forgotten or overlooked.

Reply:
We have already an issue report for the default format: #58

Reply:
Removed comment.